README.md

```diff
@@ -63,6 +63,25 @@ zipper.zipToFile ({
 });
 ```
 
+### Pipe zip data to stream (using Express.js)
+```
+app.all('/', function (request, response) {
+    response.set('content-type', 'application/zip') // optional
+    zipper.streamZipDataTo({
+        pipe: response
+        , folderName: 'myBucketFolderName'
+        , startKey: 'keyOfLastFileIZipped' // could keep null
+        , recursive, true
+        }
+        ,function (err, result) {
+            if(err)
+                console.error(err);
+            else{
+                console.log(result)
+            }
+        })
+})
+```
```

Comment:
Thanks for this, came here looking how to use streams to pipe out the zip!  Should def get this merged!